src/app-constants.js

```diff
@@ -73,8 +73,4 @@ optionalEnvVars.forEach(key => {
   if (key in process.env) AppConstants[key] = process.env[key]
 })
 
-AppConstants.VPN_PROMO_BLOCKED_LOCALES = AppConstants.VPN_PROMO_BLOCKED_LOCALES?.split(',')
```

Comment:
We should also delete this in `.env-dist` for consistency?

Reply:
Good idea!  

Though thinking on it a bit more – technically this var is still used in the legacy build.  The `app-constants.js` file I'm targeting here is specific to the refresh build.  We should definitely clean up `.env` and `env-dist` after refresh launch...